specs/cloudbeat.spec.yml

```diff
@@ -25,3 +25,9 @@ inputs:
         - "setup.template.enabled=false"
         - "-E"
         - "gc_percent=${CLOUDBEAT_GOGC:100}"
+  - name: cloudbeat/cis_k8s

```

Comment:
@fearful-symmetry thanks for the help here 🙏 , we weren't aware of this requirement. to add to this PR, I assume we should also add `cloudbeat/cis_eks` to support our [EKS configuration](https://github.com/elastic/integrations/blob/f56a8c3dbdc31c1fbba18c8a9faa2c7045fbcb92/packages/cloud_security_posture/manifest.yml#L50), I hope I don't mix things here

Reply:
Yes, @fearful-symmetry lets make sure we add `cloudbeat/cis_eks` as an input type here as well.